package.json

```
@@ -34,10 +34,11 @@
         "test": "vitest run --config ./test/vitest.config.mts",
         "test:coverage": "vitest run --config ./test/vitest.config.mts",
         "test:watch": "vitest watch --config ./test/vitest.config.mts",
-        "clean": "rimraf --glob index* devices lib converters tsconfig.tsbuildinfo",
-        "build": "rimraf --glob index* devices lib converters && tsc && cp -R dist/* .",
-        "build-watch": "tsc --watch",
-        "prepack": "pnpm run clean && pnpm run build",
+        "clean": "rimraf --glob index* coverage dist devices lib converters tsconfig.tsbuildinfo",
+        "build": "rimraf --glob index* coverage dist devices lib converters && tsc && mv dist/* .",
+        "build:watch": "tsc --watch",
+        "build-index": "node build-models-index.mjs",
```

```suggestion
        "build:index": "node build-models-index.mjs",
```

I'm also wondering wether it should be part of the `build` command

---

I went with a `-` because it isn't really a subset of `build` like the other scripts. But yea, I think we might just integrate it directly in `build` command. I needed more control when I ran the initial tests, but now it can be part of the regular `build`.